src/analysis/types.md

```diff
@@ -239,19 +225,16 @@ COW
 
 ## Internal representation
```

Comment:
Is any of the below information still valid?

Reply:
Additionally, this document states that the typedefs are stored in Sdb, but I'm not sure if that's still the case.

Reply:
Ok, I have removed the "Internal Representation" section and mention of type definitions being stored in Sdb. Also small formatting and verbiage changes.